CODEOWNERS

```diff
@@ -23,6 +23,8 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/network/rocketmq_proxy @aaron-ai @lizhanhui @lizan
 # thrift_proxy extension
 /*/extensions/filters/network/thrift_proxy @zuercher @rgs1
+# cdn_loop extension
+/*/extensions/filters/http/cdn @justin-mp @penguingao @alyssawilk
```

Comment:
I guess this extension will be in a follow up PR?

Reply:
Yes and no.

Yes, in that I have a follow-up PR with the cdn_loop filter ready to go.

No, in that the "cdn" above is a typo from before we renamed the filter from `extensions/filters/http/cdn` to `extensions/filters/http/cdn_loop`.  Give me a sec to fix that.

Reply:
Fixed!